pkgs/data/fonts/fira-code/default.nix

```diff
@@ -1,19 +1,21 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, unzip }:
 
 stdenv.mkDerivation rec {
   name = "fira-code-${version}";
-  version = "0.6";
+  version = "1.100";
 
   src = fetchurl {
-    url = "https://github.com/tonsky/FiraCode/releases/download/${version}/FiraCode-Regular.otf";
-    sha256 = "1blalxnmrxqlm5i74jhm8j29n0zsnmqi3gppxa9szjzv4x2k5s0a";
+    url = "https://github.com/tonsky/FiraCode/releases/download/${version}/FiraCode_${version}.zip";
+    sha256 = "1fprryfy8b79ncya1jzbzbxs75kvr1zwinflr8sl3d2m1r5kl03x";
   };
 
-  phases = [ "installPhase" ];
+  buildInputs = [ unzip ];
+  phases = [ "unpackPhase" "installPhase" ];
+  sourceRoot = ".";
 
   installPhase = ''
     mkdir -p $out/share/fonts/opentype
-    cp -v $src $out/share/fonts/opentype/FiraCode-Regular.otf
+    cp -r *.otf $out/share/fonts/opentype
```

Comment:
Yeah, I would appreciate that. It's not a big problem but always nice to keep the packages tidy :-)

Note, the PR was already merged so would have to do it in a new PR.

In any case, thanks for the version update!